fv3gfs/util/utils.py

```diff
@@ -50,3 +55,18 @@ def is_c_contiguous(array: Union[np.ndarray, Storage]) -> bool:
 def ensure_contiguous(maybe_array: Union[np.ndarray, Storage]) -> None:
     if isinstance(maybe_array, np.ndarray) and not is_contiguous(maybe_array):
         raise ValueError("ndarray is not contiguous")
+
+
+def assign_array(
+    left_array: Union[np.ndarray, Storage], right_array: Union[np.ndarray, Storage],
```

Comment:
Please rename these `to_array` and `from_array`, it's not as obvious these refer to the "left" and "right" sides of an equation.